backend/src/controllers/thirdPartiesController.js

```diff
@@ -0,0 +1,84 @@
+import { createHook, getBasicUserInfo, deleteHook } from '../thirdParties/github/githubApisWrapper';
+import { LOCALHOST_NG_ROCK_URL } from '../config';
+import {
+  addUsernameMapping,
+  getGithubUsername,
+} from '../thirdParties/github/githubMappingController';
+
+const getBasicInfo = token => getBasicUserInfo({ token });
```

Comment:
Is this your app's auth token?
If yes, it is better to parse your token inside a middleware and perform all validation.

Reply:
This token is the OAuth token for Github. The JWT is handled in middleware itself